docs/content/getting-started/production-environment.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Production Best Practices
+description: Best Practices of Running OpenFGA in a Production Environment
+sidebar_position: 8
+slug: /getting-started/running-in-production
+---
+
+import {
+  DocumentationNotice,
+  RelatedSection,
+} from "@components/Docs";
+
+# Running OpenFGA in Production
+
+<DocumentationNotice />
+
+The following list outlines some guidelines and best-practices for running OpenFGA in a production environment:
+
+- [Configure Authentication](./setup-openfga.mdx#configuring-authentication)
+- Enable HTTP TLS or gRPC TLS or both
+- Configure JSON as the log.format
+- [Disable the Playground](./setup-openfga.mdx#playground).
+
+## Database Recommendations
+
+To ensure good performance for OpenFGA, it is recommended that the [database](./setup-openfga.mdx#configuring-data-storage) be:
+- Used exclusively for OpenFGA and not shared with other applications. This allows scaling the OpenFGA database independently and avoiding contention with your application database. 
+- Bootstrapped and managed with the openfga migrate tool. This will ensure the appropriate indexes are created for any specific version of OpenFGA.
```

Comment:
`openfga migrate` reads a little better since it pops out and makes it more clear we're referring to a tool.

Reply:
Actually, I will like to link this to https://openfga.dev/docs/getting-started/setup-openfga#configuring-data-storage to remind the readers how to run the migration.

Reply:
Done